packaging/os/homebrew_cask.py

```diff
@@ -503,8 +525,13 @@ def main():
     if state in ('absent', 'removed', 'uninstalled'):
         state = 'absent'
 
+
+    p['install_options'] = p['install_options'] or []
+    install_options = ['--{0}'.format(install_option)
```

Comment:
this would result in double-double dashes e.g `----force` if used in examples `homebrew_cask: name=alfred state=absent install_options="--force"`right?